Source/WSTagsField.swift

```diff
@@ -109,6 +109,15 @@ open class WSTagsField: UIView {
             textField.keyboardType = newValue
         }
     }
+    
+    public var returnKeyType: UIReturnKeyType {
```

Comment:
Well, ignore my last comment. The `textField` is an internal component and it has strong dependencies that the user could disrupt and that would leave to serious  malfunctions. I will merge this!